frontend/src/components/zaehlstelle/icons/QuelleIcon.vue

```diff
@@ -7,50 +7,49 @@
         :tooltip="icon.tooltip"
     ></tooltip-with-icon>
 </template>
-<script lang="ts">
-import { Component, Prop, Vue } from "vue-property-decorator";
+<script setup lang="ts">
 import TooltipWithIcon from "@/components/zaehlstelle/icons/TooltipWithIcon.vue";
 import IconTooltip from "@/types/util/IconTooltip";
 import Quelle from "@/types/enum/Quelle";
+import { computed, ComputedRef } from "vue";
 
-@Component({
-    components: {
-        TooltipWithIcon,
-    },
-})
-export default class QuelleIcon extends Vue {
-    @Prop({ default: false }) small?: boolean;
-    @Prop({ default: false }) dense?: boolean;
-    @Prop({ default: "black" }) color?: string;
-    @Prop() quelle!: string;
+interface Props {
+    small?: boolean;
+    dense?: boolean;
+    color?: string;
+    quelle?: string;
```

Comment:
hier ist das `?` zu viel

Reply:
Ja das lag an der Fehlermeldung in der Console. Beim aufruf kann der Wert der Property undefined sein, da der Store nicht mittels eines DefaultObjects initialisiert wurde sondern mit {} as ... Hab das angepasst